package.json

```diff
@@ -97,11 +97,12 @@
   },
   "dependencies": {},
   "scripts": {
-    "bootstrap": "npm install && lerna exec npm install",
-    "install-ci": "npm ci && lerna exec npm ci",
+    "bootstrap": "npm install && lerna bootstrap --hoist react",
```

Comment:
Curious why only hoisting react but not react-dom?

Reply:
I think it doesn't matter for tests because react-dom isn't used. I tried using a glob like `--hoist 'react|react-dom|...'` but I had some other errors with that

Reply:
It's unfortunately that comments in json are not really possible because this would be a great place to have a comment…